src/common/legacy_config_opts.h

```diff
@@ -274,7 +272,6 @@ OPTION(mon_health_to_clog, OPT_BOOL)
 OPTION(mon_health_to_clog_interval, OPT_INT)
 OPTION(mon_health_to_clog_tick_interval, OPT_DOUBLE)
 OPTION(mon_health_preluminous_compat, OPT_BOOL)
-OPTION(mon_health_max_detail, OPT_INT) // max detailed pgs to report in health detail
```

Comment:
you dropped some changes in `legacy_config_opts.h` while backporting 7a23097, would you mind putting some notes in the `Conflicts` accordingly?